src/v2i-hub/MessageReceiverPlugin/src/MessageReceiverPlugin.cpp

```diff
@@ -280,6 +281,7 @@ void MessageReceiverPlugin::OnMessageReceived(routeable_message &msg)
 										ntohl(*((uint32_t*)&(bytes.data()[24]))),
 										ntohl(*((uint32_t*)&(bytes.data()[28]))));
 								sendMsg = encode(encodedSrm, srm);
+								msgPSID = 0xE0000016;
```

Comment:
There is a code smell on this line about an implicit conversion. Please check this out in the sonar scanner report.

Reply:
@paulbourelly999 Addressed